core-java-modules/core-java-io-5/src/test/java/com/baeldung/outputstreamtobytearray/OutputStreamtoByteArrayUnitTest.java

```diff
@@ -0,0 +1,56 @@
+package com.baeldung.outputstreamtobytearray;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.io.*;
+
+public class OutputStreamtoByteArrayUnitTest {
+
+    @Test
+    public void givenFileOutputStream_whenUsingFileUtilsToReadTheFile_thenReturnByteArray() throws IOException {
+        String data = "Welcome to Baeldung!";
+        String filePath = "file.txt";
+
+        try (FileOutputStream outputStream = new FileOutputStream(filePath)) {
+            outputStream.write(data.getBytes());
+        }
+
+        byte[] byteArray = FileUtils.readFileToByteArray(new File(filePath));
+        String result = new String(byteArray);
+
+        Assertions.assertEquals(data, result);

```

Comment:
Add a static import for `Assertions.assertEquals`

Reply:
Done